src/main/java/com/dabsquared/gitlabjenkins/cause/CauseData.java

```diff
@@ -421,25 +420,6 @@ public String getMergeRequestAssignee() {
         return mergeRequestAssignee;
     }
 
-    @Exported
```

Comment:
Actually this change is temporary. Will be reverted back in a similar fashion in Milestone 2 PR.

Reply:
Why is it temporary?

Reply:
Because when we will break the CauseData according to different events recieved we will have the access to GitLabApi intance in the respective causedata's, thus allowing us to get the Merge Requests. But in the current implimentation we did not have the GitLabApi intance directly availabe in the CauseData, so i shifted the getMergeRequest method to wherever it was required.